Running in wrapper mode for '254'

Running python job.

[name: "CPU-QUOTA"
int_val: 79
int_val: 24
int_val: 19
int_val: 38
int_val: 42
int_val: 54
int_val: 43
, name: "DISK"
int_val: 37
int_val: 17
int_val: 28
int_val: 26
int_val: 7
int_val: 5
int_val: 29
, name: "MEMORY"
int_val: 16
int_val: 40
int_val: 15
int_val: 35
int_val: 5
int_val: 33
int_val: 16
, name: "NET"
int_val: 26
int_val: 17
int_val: 25
int_val: 39
int_val: 20
int_val: 36
int_val: 26
]
The paramters are {u'CPU-QUOTA': array([79, 24, 19, 38, 42, 54, 43]), u'NET': array([26, 17, 25, 39, 20, 36, 26]), u'DISK': array([37, 17, 28, 26,  7,  5, 29]), u'MEMORY': array([16, 40, 15, 35,  5, 33, 16])}
{'13.56.14.158': ['haproxy', 'node-app', 'mongo'], '18.144.43.31': ['node-app', 'mongo']}
Setting cpu quota to 60000 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, NET 87040.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, DISK 14260633.6
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, MEMORY 1500000000.0
Recovering the memory utilization
Setting cpu quota to 197500 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, NET 133120.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, DISK 31037849.6
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, MEMORY 600000000.0
Recovering the memory utilization
Setting cpu quota to 47500 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, NET 128000.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, DISK 23488102.4
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, MEMORY 562500000.0
Recovering the memory utilization
Setting cpu quota to 197500 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, NET 133120.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, DISK 31037849.6
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, MEMORY 600000000.0
Recovering the memory utilization
Setting cpu quota to 47500 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, NET 128000.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, DISK 23488102.4
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, MEMORY 562500000.0
Recovering the memory utilization
Using 5 trials
INVALID EXPERIMENT TYPE: todo-app

----------------------------------------
Problem running the job:
(<type 'exceptions.SystemExit'>, SystemExit(None,), <traceback object at 0x7f40384865a8>)
Traceback (most recent call last):
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 42, in job_runner
    elif job.language == PYTHON: run_python_job(job)
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 142, in run_python_job
    result = module.main(job.id, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 104, in main
    return explore_spearmint(workload_config, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 204, in explore_spearmint
    experiment_results = run_experiment.measure_runtime(workload_config, experiment_trials)
  File "/home/ubuntu/throttlebot/src/run_experiment.py", line 37, in measure_runtime
    exit()
  File "/usr/lib/python2.7/site.py", line 375, in __call__
    raise SystemExit(code)
SystemExit: None
None
----------------------------------------
Job file reloaded.
Could not find value in output file.
Job failed in 15.36 seconds.
